docs/data_source/catalog/iceberg_catalog.md

```diff
@@ -10,12 +10,12 @@ To ensure successful SQL workloads on your Iceberg cluster, your StarRocks clust
 
 - Distributed file system (HDFS) or object storage like AWS S3, Microsoft Azure Storage, Google GCS, or other S3-compatible storage system (for example, MinIO)
 
-- Metastore like Hive metastore, AWS Glue, or REST
+- Metastore like Hive metastore, AWS Glue, or Tabular
 
   > **NOTE**
   >
   > - If you choose AWS S3 as storage, you can use HMS or AWS Glue as metastore. If you choose any other storage system, you can only use HMS as metastore.
-  > - If you use Tabular Iceberg Catalog, you must choose REST as metastore.
+  > - If you choose Tabular as metastore, you need to use the Iceberg REST catalog.
```

Comment:
maybe change "choose" to "user" for consistent word choice. Row 171 and 209 use "use".